codes/kotlin/chapter_sorting/bubble_sort.kt

```diff
@@ -29,7 +29,7 @@ fun bubbleSortWithFlag(nums: IntArray) {
         for (j in 0..<i) {
             if (nums[j] > nums[j + 1]) {
                 // 交换 nums[j] 与 nums[j + 1]
-                nums[j] = nums[j + 1].also { nums[j] = nums[j + 1] }
+                nums[j] = nums[j + 1].also { nums[j + 1] = nums[j] }
```

Comment:
I guess it is not the typical usage of `.also` semantically.

Reply:
No, this is very common usage.